### PR TITLE
Clean up some useless code

### PR DESCRIPTION
**Pull request by richvdh**

### CODE REVIEW

File: ldap_auth_provider.py [hunk 1]
```diff
@@ -13,38 +13,25 @@
...
-from pkg_resources import parse_version
-
-
 __version__ = "0.1.4"
 
 
+# ldap3 v2 changed ldap3.AUTH_SIMPLE -> ldap3.SIMPLE
 try:
-    import ldap3
-    import ldap3.core.exceptions
```
Comment (richvdh) [inline ldap_auth_provider.py:37]: given we import `ldap3` and `ldap3.core.exceptions` above, this and the `if not ldap3` below seem redundant.

File: ldap_auth_provider.py [hunk 2]
```diff
@@ -62,18 +49,11 @@ class LDAPMode(object):
 
 
 class LdapAuthProvider(object):
-    __version__ = "0.1"
```
Comment (richvdh) [inline ldap_auth_provider.py:65]: we already have a module-level __version__, so let's bin this.

Comment (clokep) [reply ldap_auth_provider.py:65]: I really appreciation how they're different too. 😄

File: ldap_auth_provider.py [hunk 3]
```diff
@@ -13,38 +13,25 @@
...
-import logging
 import synapse
 
-from pkg_resources import parse_version
-
-
 __version__ = "0.1.4"
 
 
+# ldap3 v2 changed ldap3.AUTH_SIMPLE -> ldap3.SIMPLE
```
Comment (clokep) [inline ldap_auth_provider.py:30]: It seems that setup.py requires `ldap3>=2.8`, can we just use `ldap3.SIMPLE` everywhere?

Comment (richvdh) [reply ldap_auth_provider.py:30]: why not. it seems setup.py was bumped in #57, which long post-dates this.

**Review by clokep — APPROVED** [review]

**Pull request merged**